Stanford/JumpstartEngineering/Install/Layouts/Sitewide.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * @file
+ * Configure the sitewide layout.
+ */
+
+namespace Stanford\JumpstartEngineering\Install\Layouts;
+/**
+ *
+ */
+class Sitewide extends \AbstractInstallTask {
+
+  /**
+   * Configure the sitewide layout.
+   *
+   * @param array $args
+   *   Installation arguments.
+   */
+  public function execute(&$args = array()) {
+
+
+    $context_status = variable_get('context_status', array());
```

Comment:
No comment in the original install task. Maybe @cjwest knows why she wrote this: https://github.com/SU-SOE/stanford_sites_jumpstart_engineering/blob/7.x-1.x/includes/JumpstartSitesEngineering.php#L687.